test/functional/page_objects/visualize_page.js

```diff
@@ -216,7 +216,12 @@ export function VisualizePageProvider({ getService, getPageObjects }) {
     }
 
     async selectAggregation(myString) {
-      return find.clickByCssSelector('vis-editor-agg-params:not(.ng-hide) option[label="' + myString + '"]');
+      await retry.try(async () => {
+        await find.click('visEditorAggSelect')
```

Comment:
Semicolon needed her. That's why the build is failing :)

Hey, also, thanks for being so patient and helpful!